alias/src/main/java/software/amazon/kms/alias/BaseHandlerStd.java

```diff
@@ -8,6 +8,8 @@
 import software.amazon.cloudformation.proxy.ResourceHandlerRequest;
 
 public abstract class BaseHandlerStd extends BaseHandler<CallbackContext>  {
+  protected static final int CALLBACK_DELAY_SECONDS = 60;
```

Comment:
nit: Should we fix indentation in this file?

Reply:
Fixed